pheweb/load/external_matrix.py

```diff
@@ -67,6 +72,12 @@ def run(argv):
     parser.add_argument('--ref', default="ref", action='store', type=str, help='ref column name in result files')
     parser.add_argument('--alt', default="alt", action='store', type=str, help='alt column name in result files')
     parser.add_argument('--other_fields', action='store', type=str, help='comma separated list of other column names in result files')
+    parser.add_argument('--require_match', dest="require_match", action='store_true', help='if given, require a variant to match between the sites and pheno files for it to be written')
+    parser.add_argument('--no_require_match', dest="require_match", action='store_false', help='if given, dont require a variant to match between the sites and pheno files for it to be written')
```

Comment:
What is this? Why have two switches for binary choice? Make a default and have a switch the alter that choice!